app/templates/ui/app/detail/detail.html

```diff
@@ -34,3 +33,10 @@
     </div>
   </div>
 </div>
+<div class="row">
+  <div class="col-sm-12">
+    <button class="btn btn-default" ui-sref="root.search">Cancel</button>
```

Comment:
Using `Cancel` here looks a bit funny. How about `Back`, or `Back to (looking glass icon)`?